classes/02class/exercises/c02-network09/mfreitassm/README.md

```diff
@@ -0,0 +1,114 @@
+# c02-network09
+
+## Commands Execution Output
+
+- Commands for creating security groups for below table
+
+|Security Group Name|Rule|Type|Port Range|Source|Description
+|-|-|-|-|-|-|
+|public-sg|inbound|Custom TCP Rule|22|`<your-home-ip-address>`|SSH access from home to AWS|
+|private-sg|inbound|Custom TCP Rule|22|`<public-sg-id>`|SSH access only from public instances on AWS|
+
+```
+create a security group
+aws ec2 create-security-group --group-name public-sg --description "SSH access from home to AWS" --vpc-id vpc-04426efcf5b2228a4
+{
+    "GroupId": "sg-087b8f7ebba365ec7"
+}
+
+aws ec2 authorize-security-group-ingress --group-id sg-087b8f7ebba365ec7 --protocol tcp --port 22 --cidr 101.116.206.49/32
+
+aws ec2 create-security-group --group-name private-sg --description "SSH access only from public instances on AWS" --vpc-id vpc-04426efcf5b2228a4
+{
+    "GroupId": "sg-04a749673e309f05d"
+}
+
+
+aws ec2 authorize-security-group-ingress --group-id sg-04a749673e309f05d --protocol tcp --port 22--source-group sg-087b8f7ebba365ec7
```

Comment:
```suggestion
aws ec2 authorize-security-group-ingress --group-id sg-04a749673e309f05d --protocol tcp --port 22 --source-group sg-087b8f7ebba365ec7
```

Reply:
Applied the best practice of creating SGs rules using other SGs as a source! Good one!